src/app/shared/components/product-item/product-item.component.spec.ts

```diff
@@ -1,22 +1,35 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { ProductItemComponent } from './product-item.component';
-import { ProductItemComponentVM } from './product-item.component.vm';
+import { ProductItemVM } from './product-item-vm.model';
+import { getTranslocoModule } from 'transloco-testing.module';
+
+const en = {
+  title: 'Controller',
+  description:
+    'The sleek sports car roared to life, its engine purring with power as it sped down the highway.',
+};
 
 describe('ProductItemComponent', () => {
   let component: ProductItemComponent;
   let fixture: ComponentFixture<ProductItemComponent>;
   let compiled: HTMLElement;
-  const vm: ProductItemComponentVM = {
+  const vm: ProductItemVM = {
     id: 1,
-    title: 'Controller',
-    description:
+    titleKey: 'Controller',
```

Comment:
tite**Key** and description**Key** right? But you provide the value instead of the key in the VM.